### ISSUE TITLE
TypeError: airtableJson is not a function

### ISSUE THREAD

**Issue by kbsali** [Arro/airtable-json#6]
Hi, newbie's issue here : I have copy pasted the sample code, but I'm getting the `TypeError: airtableJson is not a function` error...

```
node export-airtable-to-json.js
(node:122661) UnhandledPromiseRejectionWarning: TypeError: airtableJson is not a function
    at getStuff (file:///home/.../export-airtable-to-json.js:7:23)
    at file:///home/...export-airtable-to-json.js:15:1
    at ModuleJob.run (internal/modules/esm/module_job.js:146:23)
    at async Loader.import (internal/modules/esm/loader.js:165:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:122661) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:122661) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```
Any help/extra guidance appreciated.

Thanks!

**Comment by Arro** [comment]
I think I changed from `esm` to `core-js` module loading, which may have caused this.  Which version did it happen on?  Is it still happening?

**Comment by jmolivas** [comment]
I got the same error and did this workaround 
```
const airtableJsonOriginal = require("airtable-json");
const airtableJson = airtableJsonOriginal.default;
```

After that you can use the `airtableJson` constant as `function`
```
const getStuff = async() => {
  const songs = await airtableJson({
    auth_key, // this is your airtable api key, starting with 'key'
    base_name, // this is the base api key, which starts with 'app'
    primary: 'Songs', // this is the table name you want to pull
    view: 'Main' // this is the view you want to pull
  })

  console.log(songs)

}
```

**Comment by Arro** [comment]
Just updated deps and published to npm.  Let me know if it's still happening on version `1.3.0` .

**Comment by Arro** [comment]
FYI here, just released version `2.0.0` and it uses ESM.  You shouldn't need to do the `.default` thing anymore.
